app/stacks/global/front-door/waf-policy.tf

```diff
@@ -85,7 +302,7 @@ resource "azurerm_frontdoor_firewall_policy" "default" {
 

```

Comment:
If we are removing blocks, we can probably remove the exclusions. Or maybe comment them out for now?

Reply:
Thanks Emil, I'll comment them out.